hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -63,6 +64,7 @@ let cancelPieSignal: AbortController
 let cancelSummarySignal: AbortController
 let cancelQuotaSignal: AbortController;
 let cancelKeyMetadataSignal: AbortController;
+let adddedvaluesBlockSize: number[] = [];
```

Comment:
nit
```suggestion
let valuesWithMinBlockSize: number[] = [];
```

Reply:
@dombizita 
Done chnages in latest commit.